demo3/Demo.jsx

```diff
@@ -148,12 +149,12 @@ let Demo = React.createClass({
                           type="checkbox"
                           onChange={this.handleDone.bind(null, date)}
                           checked={isDone}
-                        />
+                          />
```

Comment:
Btw I usually keep closing bracket on the last line, so it does not look awkward.
Like:

```js
                        <button
                          className="destroy"
                          onClick={this.handleDestroy.bind(null, date)} />
```